src/benchmark/CMakeLists.txt

```diff
@@ -42,6 +42,19 @@ target_link_libraries(
     benchmark
 )
 
+add_executable(
```

Comment:
a) Should we include this?
b) Why isn't this a regular benchmark?

Reply:
Argh, this shouldn't be here. will delete it.